sklearn/metrics/_plot/tests/test_roc_curve_display.py

```diff
@@ -26,8 +26,16 @@ def data():
 
```

Comment:
I think the `data` fixture above can be removed as it is now no longer used (please double check).